src/element-service.js

```diff
@@ -203,14 +203,29 @@ export function extensionScriptsInNode(head) {
   return scripts;
 }
 
+/**
+ * Waits for body to be present then verifies that an extension script is
+ * present in head for installation.
+ * @param {!Window} win
+ * @param {HTMLHeadElement|Element|ShadowRoot} head
+ * @param {string} extensionId
+ * @return {!Promise<boolean>}
+ */
+export function isExtensionScriptInNode(win, head, extensionId) {
+  return dom.waitForBodyPromise(win.document)
```

Comment:
I don't think this will do the right thing in PWA, since `win.document.body` != `ampdoc.getBody()`. 😓

Reply:
done